src/lib.rs

```diff
@@ -269,37 +446,37 @@ mod tests {
         let sk2 = SecretId::new();
         let pk2 = sk2.public_id();
 
-        let shared_key1 = sk1.shared_key(&pk2);
-        let shared_key2 = sk2.shared_key(&pk1);
+        let shared_sk1 = sk1.shared_secret(&pk2);
+        let shared_sk2 = sk2.shared_secret(&pk1);
 
-        let ciphertext = unwrap!(shared_key1.encrypt_bytes(&data), "couldn't encrypt data");
+        let ciphertext = unwrap!(shared_sk1.encrypt_bytes(&data), "couldn't encrypt data");
         assert_ne!(&ciphertext, &data);
 
         let plaintext = unwrap!(
-            shared_key2.decrypt_bytes(&ciphertext),
+            shared_sk2.decrypt_bytes(&ciphertext),
             "couldn't decrypt data"
         );
         assert_eq!(&plaintext, &data);
 
         // Trying with wrong data
-        let error_res: Result<_, _> = shared_key2.decrypt_bytes(&plaintext);
+        let error_res: Result<_, _> = shared_sk2.decrypt_bytes(&plaintext);
```

Comment:
This (and a few other instances) could be cut down a bit to:
```rust
let _ = shared_sk2.decrypt_bytes(&plaintext).expect_err(
    "Unexpected result: we're using wrong data, it should have returned an error",
);
```

Reply:
Ah great, didn't know this was added to `Result` :+1: Thanks!